CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 0.2.2 (Next)
 
-* Your contribution here
+* [#57](https://github.com/ruby-grape/grape-swagger-rails/pull/57): Support Swagger-UI supportedSubmitMethods option - [@ShadowWrathOogles](https://github.com/ShadowWrathOogles).
 
```

Comment:
Super nitpicky (and I apologize), but could you re-add `* Your contributions here` after your addition?  After that, this is good to merge.  No need to worry about squashing, GitHub now lets me do that automatically.

Thank you again for your contribution!